### PR TITLE
[AngularJS] Tech's name fix  

### PR DESCRIPTION
**Pull request by Dorian30**
## Summary

- Changed `angular` text for `angularJS` 

## Trello Card

https://trello.com/c/hlKmaDX1

### CODE REVIEW

File: frontend/angular-js/README.md [hunk 1]
```diff
@@ -10,11 +10,11 @@
 
 ## Project Bootstrapper
 
-At [Wolox](http://wolox.com.ar), we have developed the [Angular Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our Angular projects. You'll find a detailed guide to start with your project in its README.
+At [Wolox](http://wolox.com.ar), we have developed the [AngularJS Bootstrap](http://github.com/wolox/frontend-bootstrap) (use `angular` branch) to create our AngularJS projects. You'll find a detailed guide to start with your project in its README.
```
Comment (FrankIglesias) [inline frontend/angular-js/README.md]: This is no longer the oficial angularjs bootstrap... This is the url of the new one
https://github.com/Wolox/angularjs-bootstrap

File: frontend/angular-js/README.md [hunk 2]
```diff
@@ -10,11 +10,11 @@
...
 
 ## Style Guide
 
-We use the following [Style Guide](https://github.com/johnpapa/angular-styleguide) as reference for our Angular style projects.
+We use the following [Style Guide](https://github.com/johnpapa/angular-styleguide) as reference for our AngularJS style projects.
```
Comment (FrankIglesias) [inline frontend/angular-js/README.md]: 404... 😢 
https://github.com/johnpapa/angular-styleguide

File: frontend/angular-js/README.md (hunk 1 above)
Comment (Dorian30) [reply frontend/angular-js/README.md]: I see! I'll change it

File: frontend/angular-js/README.md (hunk 2 above)
Comment (Dorian30) [reply frontend/angular-js/README.md]: @FrankIglesias I can get in to the link I already set :thinking:

**Pull request merged**